schema/model_system.json

```diff
@@ -76,6 +76,7 @@
       ]
     },
     "aux": {
+      "title": "auxiliary",
```

Comment:
"Auxiliary" in caps (output is the same, but just makes it look more like a title)